include/real_time_tools/usb_stream.hpp

```diff
@@ -171,7 +171,7 @@ class UsbStream{
    * @return std::string the debug string
    */
   static std::string msg_debug_string(const std::vector<uint8_t>& msg,
-                                      long unsigned int until=-1);
+                                      std::size_t until=-1);
```

Comment:
why does this unsigned integer type by default have a negative value?

Reply:
Ah! so, this is my mistake... Thanks for pointing this out!